tests/amt.test

```diff
@@ -0,0 +1,21 @@
+# tests for https://github.com/simonmichael/hledger/issues/457
+# note a shelltest already provides one level of quoting, so amt:>0 not 'amt:>0'
+#
+$ hledger -f - ui amt:>0
+>2 !/could not parse/
+>=1
+
+
+$ hledger -f - ui amt:<0
+>2 !/could not parse/
+>=1
+
+
+$ hledger -f - ui amt:>=0
+>2 !/could not parse/
+>=1
+
+
+$ hledger -f - ui amt:<=0
+>2 !/could not parse/
+>=1
```

Comment:
+1 for tests. There's tests/cli/cli.test, but I think tests/addons/addons.test would be the place for these.

Reply:
I have moved the tests to `tests/addons/addons.test`.

There's still the problem that the tests actually fail (even though the solution appears to work on the CLI!)